src/main/sensors/gyro.c

```diff
@@ -487,32 +497,41 @@ FAST_CODE void gyroFiltering(timeUs_t currentTimeUs)
     if (gyro.useDualGyroDebugging) {
         switch (gyro.gyroToUse) {
         case GYRO_CONFIG_USE_GYRO_1:
```

Comment:
There are enough debug slots for all gyros now - IMO all cases can be merged together too.

Reply:
Yep I would definitely agree.